smithy-model/src/main/java/software/amazon/smithy/model/traits/RetryableTrait.java

```diff
@@ -69,7 +73,8 @@ public ShapeId getShapeId() {
         @Override
         public RetryableTrait createTrait(ShapeId target, Node value) {
             ObjectNode node = value.expectObjectNode();
-            return builder().throttling(node.getBooleanMemberOrDefault(THROTTLING)).build();
+            Builder builder = builder().sourceLocation(value.getSourceLocation());
```

Comment:
This is used to create a modified Trait instance. Does continuing the sourceLocation make sense?

Reply:
Probably helps with some traceability. Most existing tratis propagate it, so updating any traits, like RetryableTrait, that aren't doing so.